eslzArm/managementGroupTemplates/policyAssignments/china/mcDINE-MDFCConfigPolicyAssignment.json

```diff
@@ -89,7 +89,7 @@
                         "value": "[parameters('logAnalyticsResourceId')]"
                     },
                     "ascExportResourceGroupName": {
-                        "value": "[concat(parameters('topLevelManagementGroupPrefix'), '-asc-export')]"
+                        "value": "[concat('rg-', parameters('topLevelManagementGroupPrefix'), '-asc-export')]"
```

Comment:
Is this not going to cause a load of remediation jobs and non-compliant resources, as this is the source of truth for TF and Bicep also ....